mmdet3d/datasets/custom_3d.py

```diff
@@ -105,7 +105,7 @@ def get_data_info(self, index):
         if not self.test_mode:
             annos = self.get_ann_info(index)
             input_dict['ann_info'] = annos
-            if self.filter_empty_gt and len(annos['gt_bboxes_3d']) == 0:
+            if self.filter_empty_gt and ~(annos['gt_labels_3d'] != -1).any():
```

Comment:
Can we check which case will cause the issue that make `gt_labels_3d=-1`? This bug should be fixed in other places.

Reply:
Class label of removed objects will be set as -1 in [here](https://github.com/open-mmlab/mmdetection3d/blob/master/mmdet3d/datasets/kitti_dataset.py#L177) . But we need to keep these Bbox for collision test.